mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxStateStore.kt

```diff
@@ -1,11 +1,11 @@
 package com.airbnb.mvrx
 
-import io.reactivex.Observable
-import io.reactivex.disposables.Disposable
+import kotlinx.coroutines.flow.Flow
 
-interface MvRxStateStore<S : Any> : Disposable {
+interface MvRxStateStore<S : Any> {
     val state: S
     fun get(block: (S) -> Unit)
     fun set(stateReducer: S.() -> S)
-    val observable: Observable<S>
+    fun cancel()
```

Comment:
it seems antithetical to coroutines to have an explicit cancel function

Reply:
See above but I also changed how this works in the latest commit